cfgov/cfgov/settings/base.py

```diff
@@ -177,7 +177,8 @@
 # Absolute path to the directory static files should be collected to.
 STATIC_ROOT = os.environ.get('DJANGO_STATIC_ROOT', '/var/www/html/static')
 
-MEDIA_ROOT = os.path.join(PROJECT_ROOT, 'f')
+MEDIA_ROOT = os.environ.get('MEDIA_ROOT', 
+                            os.path.join(PROJECT_ROOT, 'f'))
```

Comment:
@richaagarwal Ah, sorry, I misunderstood, you're talking about something different from @anselmbradford. Set the default in `os.environ` at the top of the base settings file?

So is that something that should be done for all our environment variables then, and so a separate piece of work, or is that a blocker for this PR?